### PR TITLE
KAADO-28: Implement card type service

### PR DESCRIPTION
**Pull request by nasudon**
- Implement card type service
- Use a dummy user id until user service and auth service is implemented

### CODE REVIEW

File: src/services/typeService.ts [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+        const parsedId = stringToObjectId(id);
+
+        if (!parsedId) {
+            return { error: 'Type not found' };
+        }
+        const type = await TypeModel.findOneAndDelete<IType>({ _id: parsedId, user, isDefault: false });
+        if (type) {
+            return {};
+        } else {
+            return { error: 'Type not found' };
```
Comment (tsoibet) [inline src/services/typeService.ts:136]: [NIT] I think it's better not to give false information

**Review by tsoibet — APPROVED** [review]
LGTM. You may check nitpick comment.

File: src/services/typeService.ts (hunk 1 above)
Comment (nasudon) [reply src/services/typeService.ts:136]: Updated.

**Review by tsoibet — APPROVED** [review]
LGTM

**Pull request merged**